test/uk/gov/hmrc/gform/eval/EvaluationResultsSpec.scala

```diff
@@ -357,9 +357,9 @@ class EvaluationResultsSpec extends Spec with TableDrivenPropertyChecks {
   it should "evaluate form component reference from outside, when they are in indexed pages (repeating page, ATL)" in {
 
     val table = Table(
-      ("typeInfo", "recData", "evaluationContext", "expectedResult", "scenario"),
+      ("typeInfo", "recData", "evaluationContext", "expectedResult", "exprMap", "scenario"),
       (
-        TypeInfo(FormCtx(FormComponentId("addToListNumField")), StaticTypeData(ExprType.number, None)),
+        TypeInfo(Sum(FormCtx(FormComponentId("addToListNumField"))), StaticTypeData(ExprType.number, None)),
```

Comment:
Why change to `Sum`?

Reply:
I think I replaced existing test instead of adding a new one. I will fix it.